docs/install/knative-with-operators.md

```diff
@@ -8,7 +8,9 @@ Knative provides operators as tools to install, configure and manage Knative. Th
 uninstall Knative using Knative operators.
 
 Each component in Knative has a separate operator for installation and configuration. This means that there is a [Serving operator](https://github.com/knative/serving-operator)
-and an [Eventing operator](https://github.com/knative/eventing-operator), and you can choose to install one or both independently.
+and an [Eventing operator](https://github.com/knative/eventing-operator). Knative also releases both of them together as
+the [All-In-One operator](https://github.com/knative-sandbox/operator). You can choose to install them in one bundle or
```

Comment:
Why not rewrite this to just document the all-in-one operator?

Reply:
(Which will start showing up in the main docs when we release 0.15)

Reply:
I think this could be a quick fix for release 14 branch, so that 0.14 doc can still point to the integrated operator. After this is merged, release 14 does a cherry-pick.